src/ameba/spec/expect_issue.cr

```diff
@@ -11,51 +13,96 @@ require "./util"
 #
 # Usage:
 #
-#     expect_issue subject, %(
-#       def foo
+#     expect_issue subject, <<-CRYSTAL
 #         a do
 #           b
 #         end.c
 #       # ^^^^^ error: Avoid chaining a method call on a do...end block.
-#       end
-#     )
+#       CRYSTAL
 #
 # Equivalent assertion without `expect_issue`:
 #
-#     source = Source.new %(
-#       def foo
-#         a do
-#           b
-#         end.c
-#       end
-#     ), "source.cr"
+#     source = Source.new <<-CRYSTAL, "source.cr"
+#       a do
+#         b
+#       end.c
+#       CRYSTAL
 #     subject.catch(source).should_not be_valid
 #     source.issues.size.should be(1)
 #
 #     issue = source.issues.first
-#     issue.location.to_s.should eq "source.cr:4:3"
-#     issue.end_location.to_s.should eq "source.cr:4:7"
+#     issue.location.to_s.should eq "source.cr:3:1"
+#     issue.end_location.to_s.should eq "source.cr:3:5"
 #     issue.message.should eq(
 #       "Avoid chaining a method call on a do...end block."
 #     )
 #
+# Autocorrection can be tested using `expect_correction` after
+# `expect_issue`.
+#
+#     source = expect_issue subject, <<-CRYSTAL
+#         x % 2 == 0
+#       # ^^^^^^^^^^ error: Replace with `Int#even?`.
```

Comment:
Still in the previous format, ditto all below.

Reply:
```cr
expect_issue subject, <<-CRYSTAL, keyword: keyword
    %{keyword} Exception.new(msg)
  # ^{keyword}^^^^^^^^^^^^^^^^^^^ error: Redundant `Exception.new` [...]
  CRYSTAL
```

What about this one?

Reply:
Actually... I can just delete two carets